### DISCUSSION TITLE
更新版本后错误

### DISCUSSION THREAD

**Discussion by Lachuann** [spencerwooo/onedrive-vercel-index#859] [Q&A]
更新版本后我本来部署好的就被覆盖了，然后就再也装不上了
{"error":{"code":"504","message":"An error occurred with your deployment"}}
显示这个代码

**Comment by Lachuann** [comment]
"<!DOCTYPE html><html lang=\"en\"><head><meta charSet=\"utf-8\"/><meta name=\"viewport\" content=\"width=device-width\"/><title>500: Internal Server Error</title><meta name=\"next-head-count\" content=\"3\"/><meta name=\"description\" content=\"OneDrive Vercel Index\"/><link rel=\"icon\" href=\"/favicon.ico\"/><link rel=\"preconnect\" href=\"https://fonts.googleapis.com\"/><link rel=\"preconnect\" href=\"https://fonts.gstatic.com\" crossorigin=\"\"/><link rel=\"stylesheet\" data-href=\"https://fonts.googleapis.com/css2?family=Fira+Mono&amp;family=Inter:wght@400;500;700&amp;display=swap\"/><link rel=\"preload\" href=\"/_next/static/css/fce9772795e0e66d.css\" as=\"style\"/><link rel=\"stylesheet\" href=\"/_next/static/css/fce9772795e0e66d.css\" data-n-g=\"\"/><noscript data-n-css=\"\"></noscript><script defer=\"\" nomodule=\"\" src=\"/_next/static/chunks/polyfills-c67a75d1b6f99dc8.js\"></script><script src=\"/_next/static/chunks/webpack-91bab7dd03bb5c35.js\" defer=\"\"></script><script src=\"/_next/static/chunks/framework-c16fc4c01675a4d8.js\" defer=\"\"></script><script src=\"/_next/static/chunks/main-78f82933e2573048.js\" defer=\"\"></script><script src=\"/_next/static/chunks/pages/_app-61d3e37a4e7f401e.js\" defer=\"\"></script><script src=\"/_next/static/chunks/pages/_error-4eba25a981a1d051.js\" defer=\"\"></script><script src=\"/_next/static/t_os4HqCCa1g4f0xiSwM5/_buildManifest.js\" defer=\"\"></script><script src=\"/_next/static/t_os4HqCCa1g4f0xiSwM5/_ssgManifest.js\" defer=\"\"></script><link rel=\"stylesheet\" href=\"https://fonts.googleapis.com/css2?family=Fira+Mono&family=Inter:wght@400;500;700&display=swap\"/></head><body><div id=\"__next\"><style>\n #nprogress {\n pointer-events: none;\n }\n #nprogress .bar {\n background: rgb(156, 163, 175, 0.9);\n position: fixed;\n z-index: 9999;\n top: 0;\n left: 0;\n width: 100%;\n height: 1px;\n }\n #nprogress .peg {\n display: block;\n position: absolute;\n right: 0px;\n width: 100px;\n height: 100%;\n box-shadow: 0 0 10px rgb(156, 163, 175, 0.9), 0 0 5px rgb(156, 163, 175, 0.9);\n opacity: 1;\n -webkit-transform: rotate(3deg) translate(0px, -4px);\n -ms-transform: rotate(3deg) translate(0px, -4px);\n transform: rotate(3deg) translate(0px, -4px);\n }\n #nprogress .spinner {\n display: block;\n position: fixed;\n z-index: 1031;\n top: 15px;\n right: 15px;\n }\n #nprogress .spinner-icon {\n width: 18px;\n height: 18px;\n box-sizing: border-box;\n border: solid 2px transparent;\n border-top-color: rgb(156, 163, 175, 0.9);\n border-left-color: rgb(156, 163, 175, 0.9);\n border-radius: 50%;\n -webkit-animation: nprogresss-spinner 400ms linear infinite;\n animation: nprogress-spinner 400ms linear infinite;\n }\n .nprogress-custom-parent {\n overflow: hidden;\n position: relative;\n }\n .nprogress-custom-parent #nprogress .spinner,\n .nprogress-custom-parent #nprogress .bar {\n position: absolute;\n }\n @-webkit-keyframes nprogress-spinner {\n 0% {\n -webkit-transform: rotate(0deg);\n }\n 100% {\n -webkit-transform: rotate(360deg);\n }\n }\n @keyframes nprogress-spinner {\n 0% {\n transform: rotate(0deg);\n }\n 100% {\n transform: rotate(360deg);\n }\n }\n </style><div style=\"font-family:-apple-system, BlinkMacSystemFont, Roboto, &quot;Segoe UI&quot;, &quot;Fira Sans&quot;, Avenir, &quot;Helvetica Neue&quot;, &quot;Lucida Grande&quot;, sans-serif;height:100vh;text-align:center;display:flex;flex-direction:column;align-items:center;justify-content:center\"><div><style>\n body { margin: 0; color: #000; background: #fff; }\n .next-error-h1 {\n border-right: 1px solid rgba(0, 0, 0, .3);\n }\n\n @media (prefers-color-scheme: dark) {\n body { color: #fff; background: #000; }\n .next-error-h1 {\n border-right: 1px solid rgba(255, 255, 255, .3);\n }\n }</style><h1 class=\"next-error-h1\" style=\"display:inline-block;margin:0;margin-right:20px;padding:0 23px 0 0;font-size:24px;font-weight:500;vertical-align:top;line-height:49px\">500</h1><div style=\"display:inline-block;text-align:left;line-height:49px;height:49px;vertical-align:middle\"><h2 style=\"font-size:14px;font-weight:normal;line-height:49px;margin:0;padding:0\">Internal Server Error.</h2></div></div></div></div><script id=\"__NEXT_DATA__\" type=\"application/json\">{\"props\":{\"pageProps\":{\"statusCode\":500}},\"page\":\"/_error\",\"query\":{},\"buildId\":\"t_os4HqCCa1g4f0xiSwM5\",\"nextExport\":true,\"isFallback\":false,\"gip\":true,\"locale\":\"en\",\"locales\":[\"en\",\"es\",\"zh-CN\",\"hi\",\"tr-TR\",\"zh-TW\"],\"defaultLocale\":\"en\",\"scriptLoader\":[]}</script></body></html>"

**Comment by spencerwooo** [comment]
我建议重新从头来一遍，包括配置什么的。上面贴的这个报错看不出来什么的 :(

**Reply by Lachuann** [reply]
现在没这个报错了，我用的e5订阅，现在卡在授权码那里，不改api的话能保存token然后就报错504，改了e5的api后就出现无法登录，代码如下：系统内没有 zrxz.onmicrosoft.com。请务必键入正确的域。

**Comment by Lachuann** [comment]
解决了，你那个教程挂的api有点问题
